<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="project51.qrc">
    <normaloff>:/images/circle.png</normaloff>:/images/circle.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox{
background-color:white;
border: none;
border-radius: 4px;
}
#label_2{
color:red;
}
QRadioButton
{color:white;
}
QLabel
{color:white;
}
#Frame
{background:#333;
}
QPushButton
{background:white;
color:black;
border-radius:15px;
}
QPushButton:hover
{color:#333;
border-radius:15px;
background:#49ebff;
}
QTextEdit
{background:rgb(255, 255, 255);
border-radius:15px;
}
QLineEdit
{background:transparent;
border:none;
color: white;
border-bottom:1px solid #e2e2e2;
}
#btnabout{
background:transparent;
color:grey;
}
#btnabout:hover{

color:white;
}</string>
  </property>
  <widget class="QLineEdit" name="txtuserid">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtuserpass">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnsubmit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome üòÉ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnsignup">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Signup</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnback">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnproceed">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtuserid_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtuserpass_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtcpass">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sign-up üòÅ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>or create new --&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnabout">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>about</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnproceed2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtskey">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Create Admin ID</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="project51.qrc"/>
 </resources>
 <connections/>
</ui>
